target/linux/mediatek/patches-5.15/103-mt7623-enable-arch-timer.patch

```diff
@@ -10,11 +10,11 @@ Signed-off-by: Chuanhong Guo <gch981213@gmail.com>
 
 --- a/arch/arm/mach-mediatek/Kconfig
 +++ b/arch/arm/mach-mediatek/Kconfig
-@@ -26,6 +26,7 @@ config MACH_MT6592
- config MACH_MT7623
- 	bool "MediaTek MT7623 SoCs support"
+@@ -22,6 +22,7 @@ config MACH_MT6589
+ config MACH_MT6592
+ 	bool "MediaTek MT6592 SoCs support"
```

Comment:
...the actual mt7623 patch changes different lines.

Reply:
And done too.